parsely_raw_data/event.py

```diff
@@ -32,6 +32,8 @@ def __eq__(self, other):
         mro_slots = [item for slots in
                      [a.__slots__ for a in type(self).mro() if hasattr(a, "__slots__")]
                      for item in slots]
+        # deduplicate mro slots list
+        mro_slots = list(set(mro_slots))
```

Comment:
Although, you know, using `set` instead of `dict` to keep track of things we've seen, since `set` exists now (the post is from 2008).